scripts/config/lib/nginx_config_util.rb

```diff
@@ -44,4 +44,16 @@ def self.match_redirects(redirects, uri)
 
     false
   end
+
+  def self.interpolate(string, vars)
+    regex = /\${(\w*?)}/
+
+    string.scan(regex).inject(string) do |acc, capture|
+      var_name = capture.first
+      value = vars[var_name] if vars
+      acc.sub!("${#{var_name}}", value) if value
```

Comment:
I know `\w` only matches regex-safe characters, but I'd feel happier if `var_name` were passed through `Regexp.escape` in case we ever widen the pattern.